src/SourceCode.Clay/Number.cs

```diff
@@ -875,20 +954,30 @@ decimal IConvertible.ToDecimal(IFormatProvider provider)
             }
         }
 
-        double IConvertible.ToDouble(IFormatProvider provider)
+        public double ToDouble()
+            => _double; // Same but faster result as calling ToDouble(CultureInfo.InvariantCulture)
+
+        public double ToDouble(IFormatProvider provider)
```

Comment:
Not hidden behind explicit interface since it's a lowest-common-denominator method